### PR TITLE
#207 - ParameterCount should ignore methods with @Override annotation

### PR DESCRIPTION
**Pull request by ychulovskyy**

### CODE REVIEW

File: src/main/groovy/org/codenarc/rule/size/ParameterCountRule.groovy [hunk 1]
```diff
@@ -17,8 +17,8 @@ package org.codenarc.rule.size
 
 import org.codehaus.groovy.ast.ConstructorNode
 import org.codehaus.groovy.ast.MethodNode
-import org.codenarc.rule.AbstractAstVisitorRule
 import org.codenarc.rule.AbstractAstVisitor
+import org.codenarc.rule.AbstractAstVisitorRule
```
Comment (ychulovskyy) [inline src/main/groovy/org/codenarc/rule/size/ParameterCountRule.groovy]: Formatted with default IntelliJ Idea formatter

File: src/test/groovy/org/codenarc/rule/size/ParameterCountRuleTest.groovy [hunk 2]
```diff
@@ -189,6 +189,65 @@ class ParameterCountRuleTest extends AbstractRuleTestCase {
...
+            class TestClass {
+
+                TestClass() {
+                }
+
+                void someMethod() {
+                }
+
+                @Override
+                void someMethod(int arg1, int arg2, int arg3, int arg4, int arg5) { ${violation('method TestClass.someMethod')}
```
Comment (chrismair) [inline src/test/groovy/org/codenarc/rule/size/ParameterCountRuleTest.groovy]: Can you remove the ${violation... ?  It is confusing.

**Comment by chrismair** [comment]
Thanks for the contribution!

**Comment by chrismair** [comment]
I changed *ignoreOverriddenMethods* to default to true. I think that in most cases that is reasonable default behavior. And since that is reducing the number of violations rather than adding new ones, it should not cause *breaking changes* for anyone.

**Pull request merged**